docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_extended_security_updates/_index.md

```diff
@@ -0,0 +1,173 @@
+---
+type: docs
+title: "Using Azure Arc to deliver Extended Security Updates for Windows Server and SQL Server 2012"
+linkTitle: "Using Azure Arc to deliver Extended Security Updates for Windows Server and SQL Server 2012"
+weight: 18
+description: >
+---
+
+## Using Azure Arc to deliver Extended Security Updates for Windows Server and SQL Server 2012
+
+The following Jumpstart scenario will guide you on how to use Azure Arc to enroll Windows Server and SQL Server 2012/2012 R2 machines in [Extented Security Updates (ESUs)](https://learn.microsoft.com/windows-server/get-started/extended-security-updates-overview). This scenario creates an Azure VM with Hyper-V installed where the Windows Server and/or and SQL Server VMs will run and will be onboarded to Azure Arc-enabled server and Azure Arc-enabled SQL server respectively. Once these VMs are registered in Azure you will have visibility into their ESU coverage and will be able to enroll them through the Azure portal one month before Windows Server 2012 end of support.
+
+In this scenario you can choose between working with Windows Server, SQL Server 2012 R2 or both.
```

Comment:
It's not really accurate. 

The base image is Windows Server 2012 R2 and the way it's written it looks like SQL is R2. 
- Fix the Windows Server to R2
- SQL is "Standard" edition, worth calling this out.